packages/grid/default.env

```diff
@@ -95,6 +95,7 @@ JAEGER_PORT=14268
 
 # Syft
 SYFT_TUTORIAL_MODE=False
+ENABLE_WARNINGS=True
```

Comment:
Q: Since by orchestra , by default , enable warnings is False, should we set here also to False?Q

Reply:
Made it false for orchestra by default because its used in test and notebooks, where we want it to be false